tests/utils/verification/volumeproperties.go

```diff
@@ -97,6 +100,75 @@ func GetVolumePropertiesDockerCli(volName string, hostname string) string {
 	return op
 }
 
+// GetContainerName returns full container name based on given short name
+func GetContainerName(hostName, shortName string) (string, error) {
+	cmd := dockercli.ListContainers + "--filter name='" + shortName + "' --format '{{.Names}}'"
+	return ssh.InvokeCommand(hostName, cmd)
+}
+
+// GetAllContainers returns all running containers on the give host based on the filtering criteria
+func GetAllContainers(hostName, filter string) (string, error) {
+	cmd := dockercli.ListContainers + "--filter name='" + filter + "' --format '{{.Names}}'"
+	return ssh.InvokeCommand(hostName, cmd)
+}
+
+// VerifyDockerService returns true if the given service is running on
+// the specified VM with specified replicas; false otherwise.
+func VerifyDockerService(hostName, serviceName string, replicas int) bool {
+	log.Printf("Checking docker service [%s] status on VM [%s]\n", serviceName, hostName)
+	out, err := util.ListService(hostName, serviceName)
+	if err != nil {
+		log.Println(err)
+		return false
+	}
+
+	for i := 1; i <= replicas; i++ {
+		if !strings.Contains(out, serviceName+"."+strconv.Itoa(i)) {
+			return false
+		}
+	}
+	return true
+}
+
+// VerifyDockerContainer returns true and the host IP if the containers are running on one of
+// the given docker hosts with specified replicas; otherwise returns false and empty string.
+func VerifyDockerContainer(dockerHosts []string, serviceName string, replicas int) (bool, string) {
+	log.Printf("Checking running containers for docker service [%s] on docker hosts: %v\n", serviceName, dockerHosts)
+
+	//TODO: Need to implement generic polling logic for better reuse
+	for attempt := 0; attempt < maxAttempt; attempt++ {
```

Comment:
I would suggest to move out this code block into another method and put in place at `misc`.